scripts/setup/setup-reportstream.sh

```diff
@@ -16,6 +16,18 @@ echo "Resetting the database and loading the baseline settings..."
 ./gradlew reloadTables
 ./gradlew reloadSettings
 
+# Update RS Configuration for the TI API based on docker or gradle
+# If either TI or RS are not running in docker, then use a gradle hosted URL
+LOCAL_DOCKER_IMAGE_NAME=$(docker ps --filter "name=trusted-intermediary-router-1" | grep trusted-intermediary-router-1)
```

Comment:
Naming convention for local variables is to use lowercase. Could you also add `ti` to this variable name? So the variables names would be `local_ti_docker_image_name` and `local_rs_docker_image_name`